packages/core-utils/src/route.ts

```diff
@@ -207,7 +208,8 @@ const routeTypeComparatorValue = {
   // TODO: 11 and 12 are not a part of OTP as of 2019-02-14, but for now just
   // associate them with bus/rail.
   11: modeComparatorValue.BUS, // - Trolleybus.
-  12: modeComparatorValue.RAIL // - Monorail.
+  12: modeComparatorValue.RAIL, // - Monorail.
+  13: modeComparatorValue.TROLLEYBUS
```

Comment:
Lines 210 and 212 both seem to be accounting for Trolleybuses here?

Reply:
The comment is wrong, but only 212 accounts for trolleybus

Reply:
Could we remove the wrong comment

Reply:
I'm worried about what information we might lose... These comments are very old!